src/htmx.js

```diff
@@ -403,7 +403,7 @@ return (function () {
             while(fragment.childNodes.length > 0){
                 var child = fragment.firstChild;
                 parentNode.insertBefore(child, insertBefore);
-                if (child.nodeType !== Node.TEXT_NODE) {
+                if (child.nodeType !== Node.TEXT_NODE && child.nodeType !== Node.COMMENT_NODE) {
```

Comment:
great catch!

Reply:
I guess it's not a problem because later calls to queryselector won't find anything but with this we can avoid some function calls :)

Reply:
i wonder if we should push the nodeType check inside of makeAjaxLoadTask() so we don't have to repeat ourselves...